### PR TITLE
Add noqa support

### PR DESCRIPTION
**Pull request by okhaliavka**
- Add "noqa" support
- Slightly improve wording

### CODE REVIEW

File: flake8_sorted_keys/checker.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 __version__ = '0.1.0'
 
 import ast
+import re
+
+NOQA_REGEX = re.compile(r'#\s*noqa(?=(:.*?S001|$))', re.I)
```
Comment (yevhen-m) [inline flake8_sorted_keys/checker.py]: Verbose mode please.

File: flake8_sorted_keys/checker.py [hunk 2]
```diff
@@ -9,16 +12,25 @@ class SortedKeysChecker(object):
...
 
     def needs_checking(self, dict_node):
-        """Decide weather specific Dict literal node needs to be considered.
+        """Decide if specific Dict literal node needs to be considered.
 
         We are only interested in multi-line dicts with all string keys.
         """
         if not all(isinstance(key, ast.Str) for key in dict_node.keys):
             return False
+        if self.noqa(dict_node):
```
Comment (yevhen-m) [inline flake8_sorted_keys/checker.py]: Newline please.

**Review by yevhen-m — CHANGES_REQUESTED** [review]

File: flake8_sorted_keys/checker.py [hunk 3]
```diff
@@ -9,16 +15,26 @@ class SortedKeysChecker(object):
...
+    def noqa(self, dict_node):
+        """Check if dict contains noqa comment."""
+        # ast enumerates lines starting from 1
+        return NOQA_REGEX.search(self.lines[dict_node.lineno - 1]) is not None
 
     def needs_checking(self, dict_node):
-        """Decide weather specific Dict literal node needs to be considered.
+        """Decide if specific Dict literal node needs to be considered.
 
         We are only interested in multi-line dicts with all string keys.
```
Comment (eprykhodko) [inline flake8_sorted_keys/checker.py]: please extend this doc to reflect your change

File: flake8_sorted_keys/checker.py [hunk 4]
```diff
@@ -9,16 +15,26 @@ class SortedKeysChecker(object):
...
         """
         if not all(isinstance(key, ast.Str) for key in dict_node.keys):
             return False
+        
+        if self.noqa(dict_node):
```
Comment (eprykhodko) [inline flake8_sorted_keys/checker.py]: no need for separate function, inline the check

**Review by eprykhodko — CHANGES_REQUESTED** [review]

**Review by eprykhodko — APPROVED** [review]

**Pull request merged**